File: src/main/java/com/elchologamer/userlogin/command/sub/ReloadCommand.java
package com.elchologamer.userlogin.command.sub;

import com.elchologamer.userlogin.ULPlayer;
import com.elchologamer.userlogin.command.base.SubCommand;
import org.bukkit.command.CommandSender;
import org.bukkit.entity.Player;

public class ReloadCommand extends SubCommand {
    public ReloadCommand() {
        super("reload", "ul.reload");
    }

    @Override
    public boolean run(CommandSender sender, String[] args) {
        getPlugin().reloadPlugin();

        for (Player player : getPlugin().getServer().getOnlinePlayers()) {
            ULPlayer ulPlayer = ULPlayer.get(player);
            if (!ulPlayer.isLoggedIn()) {
                ulPlayer.schedulePreLoginTasks();
            }
        }

        sender.sendMessage(getPlugin().getLang().getMessage("commands.reload"));
        return true;
    }
}

File: src/main/kotlin/com/elchologamer/userlogin/command/sub/ReloadCommand.kt
package com.elchologamer.userlogin.command.sub

import com.elchologamer.userlogin.ULPlayer
import com.elchologamer.userlogin.UserLogin.Companion.plugin
import com.elchologamer.userlogin.command.base.SubCommand
import com.elchologamer.userlogin.manager.LangManager
import org.bukkit.command.CommandSender

class ReloadCommand : SubCommand("reload", "ul.reload") {
    override fun run(sender: CommandSender, args: Array<String>): Boolean {
        plugin.reloadPlugin()

        for (player in plugin.server.onlinePlayers) {
            val ulPlayer = ULPlayer[player]
            if (!ulPlayer.loggedIn) {
                ulPlayer.schedulePreLoginTasks()
            }
        }

        sender.sendMessage(LangManager.getMessage("commands.reload"))
        return true
    }
}